docs/experiments-msmarco-passage.md

```diff
@@ -236,6 +239,14 @@ python tools/scripts/msmarco/convert_msmarco_to_trec_qrels.py \
   --output collections/msmarco-passage/qrels.dev.small.trec
 ```
 
+If you are using the `trec_eval` tool for the first time, you need to extract and build it first:
```

Comment:
Do we want this instruction explicitly here? it's part of the getting started right?

Reply:
> Do we want this instruction explicitly here? it's part of the getting started right?

had missed it, removed.